conf/config.ini

```diff
@@ -10,8 +10,8 @@
 ; Can be removed; defaults to HOME_DIR/.kresus.
 ; Overriden by the KRESUS_DIR environment variable, if it's set.
 ; Example:
-; datadir=/home/ben/.kresus
-datadir=__FINALPATH__/data/
+; data_dir=/home/ben/.kresus
+data_dir=__INSTALL_DIR__/data/
```

Comment:
```suggestion
datadir=__INSTALL_DIR__/data/
```

Reply:
funny... that is the second time I see the `packaging_v2` script changing `datadir` to `data_dir`